assets/css/index.css

```diff
@@ -17,7 +17,21 @@ footer {
 }
 
 .sidebar-list a {
-  display: block;
+  overflow-y: auto;
```

Comment:
not to `.sidebar-list a` just to `.sidebar-list` and yes inside media for max-width 767px

Reply:
if I do that, it will be like this
![Screenshot_2](https://user-images.githubusercontent.com/24585708/93670749-e07dd400-fac7-11ea-9626-6af37f04536a.png)
![Screenshot_1](https://user-images.githubusercontent.com/24585708/93670752-e2e02e00-fac7-11ea-8c0a-b39ed15bb81d.png)

Reply:
Keep `display: block;` for `.sidebar-list a` but move `overflow-y: auto` to just `.sidebar-list`.  
Basically, it should be something like this:

```
.sidebar-list a {
display: block;
}

@media screen and (max-width: 767px) {
  .sidebar-list {
   overflow-y: auto;
  }
}
```